src/java/org/apache/cassandra/transport/PreV5Handlers.java

```diff
@@ -318,13 +325,22 @@ public void exceptionCaught(final ChannelHandlerContext ctx, Throwable cause)
                 if (isFatal(cause))
                     future.addListener((ChannelFutureListener) f -> ctx.close());
             }
-            
-            if (DatabaseDescriptor.getClientErrorReportingExclusions().contains(ctx.channel().remoteAddress()))
+
+            SocketAddress remoteAddress = ctx.channel().remoteAddress();
+            AuthenticationException authenticationException = maybeExtractAndWrapAuthenticationException(cause);
```

Comment:
hmm, I don't know actually. Let me look into that

Reply:
We were looking at this in 1:1 chat and we saw that `PipelineConfigurator.configureInitialPipeline` https://github.com/apache/cassandra/blob/trunk/src/java/org/apache/cassandra/transport/PipelineConfigurator.java#L249-L255 wires up `PreV5Handlers.ExceptionHandler.instance` initially and then it gets swapped out in response to a Startup message handling https://github.com/apache/cassandra/blob/trunk/src/java/org/apache/cassandra/transport/InitialConnectionHandler.java#L124.

Because this is handling failures happening during a handshake, the PostV5 handler would never be in place to hit this.